### ISSUE TITLE
Editorial accidental styling

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#669]
Adding `@func="edit"` to `<accid>` will mark the accidental as an editorial accidental:

<img width="259" alt="screen shot 2017-08-18 at 5 54 32 pm" src="https://user-images.githubusercontent.com/3487289/29479098-4d0a7334-843e-11e7-91e3-a09c7b449a1f.png">

```xml
<note dur="1" oct="4" pname="c">
     <accid accid="s" func="edit" />
</note>
```

This form of editorial accidental is useful for vocal music from the Renaissance.  

However, there are two other forms of editorial accidental styles which would be useful to support.

(1) editorial accidental via bracket:

Brackets can be added to accidentals in verovio:

<img width="314" alt="screen shot 2017-08-18 at 6 14 04 pm" src="https://user-images.githubusercontent.com/3487289/29479582-09aa9c38-8441-11e7-9e12-d3c25dc1555f.png">

```xml
<note dur="1" oct="4" pname="c">
      <accid accid="s" enclose="brack"  />
</note>
```

But note that if `@enclose` and `@func="edit"` are active at the same time, the editorial accidental is above the note:

<img width="353" alt="screen shot 2017-08-18 at 6 15 57 pm" src="https://user-images.githubusercontent.com/3487289/29479615-51e8e4aa-8441-11e7-9a59-cec18dba5521.png">

```xml
<note dur="1" oct="4" pname="c">
     <accid accid="s" enclose="brack" func="edit" />
</note>
```

This graphical rendering is not needed, and instead the accidental should appear before the note with brackets rather than small and above (that would be some strange double-editorial accidental for renaissance music).

(2) Editorial accidentals indicated by parentheses:

Likewise for `@enclose="paren"`:

<img width="306" alt="screen shot 2017-08-18 at 6 18 22 pm" src="https://user-images.githubusercontent.com/3487289/29479667-a1ac4ad6-8441-11e7-8c98-8cdd0f1e2535.png">

```xml
<note dur="1" oct="4" pname="c">
      <accid accid="s" enclose="paren" func="edit" />
</note>
```

Which should instead be rendered as:

<img width="305" alt="screen shot 2017-08-18 at 6 20 04 pm" src="https://user-images.githubusercontent.com/3487289/29479710-dd2c6532-8441-11e7-8367-9c78b30b378c.png">

-------

In other words, when `@enclose` is given, `@func="edit"` should not move the accidental above the notehead and make it smaller.

**Comment by craigsapp** [comment]
Looks great:

<img width="1574" alt="screen shot 2017-09-30 at 9 55 55 am" src="https://user-images.githubusercontent.com/3487289/31047776-9b080e0e-a5c5-11e7-9b8d-2abbdc2df6d9.png">

```
**kern
=1-
4c--i
4d-i
4eni
4f#i
4g##i
=2
4c--j
4d-j
4enj
4f#j
4g##j
=3
4c--k
4d-k
4enk
4f#k
4g##k
=4
1ff--i
1ff-i
1ffni
1ff#i
1ff##i
==
*-
!!!RDF**kern: i = editorial accidental
!!!RDF**kern: j = editorial accidental paren
!!!RDF**kern: k = editorial accidental brack
```
